app/components/Card/Card.css

```diff
@@ -20,5 +15,6 @@ html[data-theme='dark'] .card {
 }
 
 .shadow {
-  box-shadow: 0 1px 4px rgba(var(--rgb-min), var(--rgb-min), var(--rgb-min), 4%);
+  box-shadow: 0 2px 8px 2px rgba(104, 112, 118, 7%),
+    0 2px 4px -1px rgba(104, 112, 118, 4%);
 }
```

Comment:
how is this on dark-mode now?

Reply:
<img width="290" alt="image" src="https://user-images.githubusercontent.com/69514187/215346335-72447593-3338-4f33-bf7b-a8b6ad520bbe.png">

Not very visible, but I kinda prefer that on dark mode. A shadow isn't really needed. That's usually how it's done in the "industry".

Reply:
I'm planning on standardizing the box-shadows across the webapp soon.